services/wallet/requests/router_input_params.go

```diff
@@ -0,0 +1,199 @@
+				return ErrENSRegisterMainnetSNTOnly
+			}
+		}
+		return nil
+	}
+
+	if i.SendType == sendtype.ENSRelease {
+		if i.Username == "" {
+			return ErrENSReleaseRequiresUsername
+		}
+	}
+
+	if i.SendType == sendtype.ENSSetPubKey {
+		if i.Username == "" || i.PublicKey == "" {
+			return ErrENSSetPubKeyRequiresUsernameAndPubKey
+		}
+
+		if ens.ValidateENSUsername(i.Username) != nil {
+			return ErrENSSetPubKeyInvalidUsername
+		}
+	}
+
+	if i.SendType == sendtype.StickersBuy {
+		if i.PackID == nil {
+			return ErrStickersBuyRequiresPackID
+		}
+	}
+
+	if i.SendType == sendtype.Swap {
+		if i.ToTokenID == "" {
+			return ErrSwapRequiresToTokenID
+		}
+		if i.TokenID == i.ToTokenID {
+			return ErrSwapTokenIDMustBeDifferent
+		}
+
+		if i.AmountIn != nil &&
+			i.AmountOut != nil &&
+			i.AmountIn.ToInt().Cmp(pathprocessor.ZeroBigIntValue) > 0 &&
+			i.AmountOut.ToInt().Cmp(pathprocessor.ZeroBigIntValue) > 0 {
+			return ErrSwapAmountInAmountOutMustBeExclusive
+		}
+
+		if i.AmountIn != nil && i.AmountIn.ToInt().Sign() < 0 {
+			return ErrSwapAmountInMustBePositive
+		}
+
+		if i.AmountOut != nil && i.AmountOut.ToInt().Sign() < 0 {
+			return ErrSwapAmountOutMustBePositive
+		}
+	}
+
+	return i.validateFromLockedAmount()
+}
+
+func (i *RouteInputParams) validateFromLockedAmount() error {
+	if i.FromLockedAmount == nil || len(i.FromLockedAmount) == 0 {
+		return nil
+	}
+
+	var suppNetworks map[uint64]bool
+	if i.TestnetMode {
+		suppNetworks = walletCommon.CopyMapGeneric(walletCommon.SupportedTestNetworks, nil).(map[uint64]bool)
+	} else {
+		suppNetworks = walletCommon.CopyMapGeneric(walletCommon.SupportedNetworks, nil).(map[uint64]bool)
+	}
+
+	if suppNetworks == nil {
+		return ErrCannotCheckLockedAmounts
+	}
+
+	totalLockedAmount := big.NewInt(0)
+	excludedChainCount := 0
+
+	for chainID, amount := range i.FromLockedAmount {
+		if walletCommon.ArrayContainsElement(chainID, i.DisabledFromChainIDs) {
+			return ErrDisabledChainFoundAmongLockedNetworks
+		}
+
+		if i.TestnetMode {
+			if !walletCommon.SupportedTestNetworks[chainID] {
+				return ErrLockedAmountNotSupportedForNetwork
+			}
+		} else {
+			if !walletCommon.SupportedNetworks[chainID] {
+				return ErrLockedAmountNotSupportedForNetwork
+			}
+		}
+
+		if amount == nil || amount.ToInt().Sign() < 0 {
+			return ErrLockedAmountNotNegative
+		}
+
+		if !(amount.ToInt().Sign() > 0) {
+			excludedChainCount++
+		}
+		delete(suppNetworks, chainID)
+		totalLockedAmount = new(big.Int).Add(totalLockedAmount, amount.ToInt())
+	}
+
+	if (!i.TestnetMode && excludedChainCount == len(walletCommon.SupportedNetworks)) ||
+		(i.TestnetMode && excludedChainCount == len(walletCommon.SupportedTestNetworks)) {
+		return ErrLockedAmountExcludesAllSupported
+	}
+
+	if totalLockedAmount.Cmp(i.AmountIn.ToInt()) > 0 {
+		return ErrLockedAmountExceedsTotalSendAmount
+	} else if totalLockedAmount.Cmp(i.AmountIn.ToInt()) < 0 && len(suppNetworks) == 0 {
+		return ErrLockedAmountLessThanSendAmountAllNetworks
+	}
```

Comment:
No need for `else` after `return`
```suggestion
	if totalLockedAmount.Cmp(i.AmountIn.ToInt()) > 0 {
		return ErrLockedAmountExceedsTotalSendAmount
	}  
	if totalLockedAmount.Cmp(i.AmountIn.ToInt()) < 0 && len(suppNetworks) == 0 {
		return ErrLockedAmountLessThanSendAmountAllNetworks
	}
```